src/schema/v2/me/collection.ts

```diff
@@ -85,6 +85,13 @@ export const CollectionType = new GraphQLObjectType<any, ResolverContext>({
       type: new GraphQLNonNull(GraphQLString),
       description:
         "Name of the collection. Has a predictable value for 'standard' collections such as Saved Artwork, My Collection, etc. Can be provided by user otherwise.",
+      resolve: ({ name }) => {
+        if (name === "Saved Artwork") {
+          return "All Saves"
+        }
+
+        return name
+      },
```

Comment:
**question**: should we add an argument and based on it return either the original name or the modified one?

Reply:
```javascript
name: {
  type: new GraphQLNonNull(GraphQLString),
  description:
    "Name of the collection. Has a predictable value for 'standard' collections such as Saved Artwork, My Collection, etc. Can be provided by user otherwise.",
  args: {
    defaultCollectionName: {
      type: new GraphQLNonNull(GraphQLString),
      description:
        "pass `MODIFIED` to return `All Saves` instead of `Saved Artwork`",
    },
  },
  resolve: ({ name }, { defaultCollectionName }) => {
    if (name === "Saved Artwork" && defaultCollectionName === "MODIFIED") {
      return "All Saves"
    }

    return name
  },
}
```

Reply:
https://user-images.githubusercontent.com/3513494/225419765-b1e173cb-72f0-48c3-92b0-86c56b772665.mp4

Reply:
It's a good thought, but I don't think it will be necessary. This `me.collection` MP field was only added recently, and our new code in Force is its only consumer.